lib/bridge/MatrixHandler.js

```diff
@@ -451,6 +451,66 @@ MatrixHandler.prototype._onAdminMessage = Promise.coroutine(function*(req, event
         yield this.ircBridge.sendMatrixAction(adminRoom, botUser, notice, req);
         return;
     }
+    else if (cmd === "!storepass") {
+        let domain = ircServer.domain;
+        let userId = event.user_id;
+        let notice;
+
+        try {
+            // Allow passwords with spaces
+            let pass = args.join('');
+
+            if (pass.length === 0) {
+                notice = new MatrixAction(
+                    "notice",
+                    "Format: '!storepass password' " +
+                    "or '!storepass irc.server.name password'\n"
```

Comment:
Explanation of what this does would be good. I would move the blurb you have on success ("When you next reconnect...") to a `var` and then use that here in addition to on success.